src/scripts/deploy-script.ts

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env node
+import {
+  AccountRegistry,
+  Chain,
+  ContractRegistryLoader,
+  deployToChain,
+  parseObjFromFile,
+} from "..";
+
+import { getMainLogger } from "../utils/cli";
+import { DEFAULT_RPC_URL } from "../utils/constants";
+
+// can either parse
+const accountConfigFromYaml = {
+  name: "local",
+  registry: parseObjFromFile("specs/evm.accounts.yaml"),
+};
+
+// Or can parse it form the env
+const accountConfig = {
+  name: "local",
+  registry: [
+    {
+      name: "KEY_POLYMER",
+      privateKey: process.env.KEY_POLYMER
+        ? process.env.KEY_POLYMER
+        : accountConfigFromYaml.registry[0].privateKey,
+    },
+  ],
+};
+
+const accounts = AccountRegistry.loadMultiple([accountConfig]);
+const contracts = ContractRegistryLoader.loadSingle(
+  parseObjFromFile("specs/contracts.spec.yaml")
+);
+
+const chain: Chain = {
+  rpc: process.env.RPC_URL ? process.env.RPC_URL : DEFAULT_RPC_URL,
+  chainId: process.env.DEPLOYMENT_CHAIN_ID
+    ? parseInt(process.env.DEPLOYMENT_CHAIN_ID)
+    : 31337,
```

Comment:
what does this number mean?

Reply:
this is the default chain ID used by hardhat (and maybe anvil)

Reply:
🙏